lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -90,22 +91,42 @@ sub _get_ptero_execute_method {
 
     my $self = shift;
     my $log_dir = shift;
+    my $ptero_lsf_parameters = $self->_get_ptero_lsf_parameters();
+    $ptero_lsf_parameters->{command} = sprintf(
+        'genome ptero wrapper --command-class %s '
+        .'--method execute --log-directory %s',
+        $self->command, $log_dir);
```

Comment:
Maybe all we need is to call `Genome::Sys->quote_for_shell` on the `log_directory`?  (This code is some layer above PTero!)